File: first_example/Log/Log.java
public class Log {
    public void debug(String message) {
        print("DEGUB ", message);
    }

    public void info(String message) {
        print("INFO ", message);
    }

    public void warning(String message) {
        print("WARNING ", message);
    }

    public void error(String message) {
        print("ERRROR ", message);
    }

    public void fatal(String message) {
        print("Fatal ", message);
        System.exit(1);
    }

    public void print(String message, String severity) {
        System.out.println(severity + ": " + message);
    }

    public static void main(String[] args) {
        Log log = new Log();
        log.debug("Test");
    }
}

File: first_example/Log/Log.py
import sys


class Log:
    def debug(self, message: str):
        self.print_msg("DEGUB ", message)

    def info(self, message: str):
        self.print_msg("INFO ", message)

    def warning(self, message: str):
        self.print_msg("WARNING ", message)

    def error(self, message: str):
        self.print_msg("ERRROR ", message)

    def fatal(self, message: str):
        self.print_msg("Fatal ", message)
        sys.exit(1)

    def print_msg(self, message: str, severity: str):
        print(severity + ": " + message)


if __name__ == '__main__':
    log = Log()
    log.debug('test')
